dot/parachain/availability-store/availability_store_test.go

```diff
@@ -986,8 +986,10 @@ func (h *testHarness) processMessages() {
 	for {
 		select {
 		case msg := <-h.overseer.SubsystemsToOverseer:
+			//if h.processes != nil && processIndex < len(h.processes) {
```

Comment:
is this meant to be commented out?

Reply:
All the changes in this file will not be included in this PR.

It's @edwardmack's changes. He will raise separate PR for this.
My PR will be merged after Ed's PR merge